docs/api/app.md

```diff
@@ -760,6 +760,23 @@ Disables hardware acceleration for current app.
 
 This method can only be called before app is ready.
 
+### `app.getAppMemoryInfo()`
+
+Returns `Object[]`:
+
+* `pid` Integer - The process id for which memory info is collected for
+  * `workingSetSize` Integer - The amount of memory currently pinned to actual physical
+    RAM.
+  * `peakWorkingSetSize` Integer - The maximum amount of memory that has ever been pinned
+    to actual physical RAM.
+  * `privateBytes` Integer - The amount of memory not shared by other processes, such as
+    JS heap or HTML content.
+  * `sharedBytes` Integer - The amount of memory shared between processes, typically
+    memory consumed by the Electron code itself
+
+Returns an object giving memory usage statistics about all the processes associated with 
+the app. Note that all statistics are reported in Kilobytes.
```

Comment:
It seems weird to me that shared**Bytes** represents kilobytes instead of bytes. Is there a reason for that? If not, it might make more sense to just use bytes and perform the division in the app code.